src/setupTests.ts

```diff
@@ -2,6 +2,21 @@ import '@testing-library/jest-dom';
 import 'blob-polyfill';
 
 import { toHaveNoViolations } from 'jest-axe';
+import failOnConsole from 'jest-fail-on-console';
+
+// Fail tests that produce console logs.
+// Console warnings or errors suggest there are issues with the test.
+// Other console logs are noise that make it harder to find informative output when tests do fail.
+if (!process.env.ALLOW_LOGS) {
```

Comment:
Please make sure to add this to the wiki.

Reply:
Added to https://github.com/DataBiosphere/terra-ui/wiki/Getting-started#testing.